infrastructure/zk/src/init.ts

```diff
@@ -67,8 +67,8 @@ export async function reinit() {
     await announced('Setting up containers', up());
     await announced('Compiling JS packages', run.yarn());
     await announced('Compile l2 contracts', compiler.compileAll());
-    await announced('Drop postgres db', db.drop());
-    await announced('Setup postgres db', db.setup());
+    await announced('Drop postgres db', db.drop({}));
+    await announced('Setup postgres db', db.setup({}));
```

Comment:
We have plenty of implicit behavior in `zk` and it has proven to cause problems. I think it's better to require everything to be passed explicitly, even at the cost of being more verbose.